codecov.yml

```diff
@@ -1,7 +1,54 @@
 coverage:
+  precision: 2
```

Comment:
Looks like cosmos already havea codecov file but it was `.codecov.yml`, just going to reuse it